CHANGELOG

```diff
@@ -1,3 +1,16 @@
+2019-03-15     1.0.0
+--------------------
+
+Enhancements:
+
+* Add ability to parse package metadata version 2.1 (#312)
+
+bug fixes:
```

Comment:
These items look that should be part of the 0.17.4 changelog. Please add the 0.17.4 section as it appears here. https://github.com/enthought/okonomiyaki/commit/658b66145ff5ceed3887205bfe6f2e95f8352824#diff-d3bb3391c79904494c60ee2ac2f33070